ansible_collections/arista/avd/docs/pyavd.md

```diff
@@ -86,6 +86,12 @@ Optional `md-toc` requirement (automatically installed with above command):
 md-toc>=8.1.8
 ```
 
+To install ansible [avd collection additional python requirements](installation/collection-installation.md#additional-python-libraries-required) install with extra `ansible`:
+
+```sh
+pip3 install pyavd[ansible]
+```
+
```

Comment:
```suggestion

This is useful as a shortcut to install all Python requirements for the `arista.avd` Ansible collection.

```

Reply:
pyavd[ansible] will install PyAVD plus the `arista.avd` collection requirements if I'm not mistaken, I'd remove the last sentence.

Reply:
OK, my bad.